wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -26,7 +26,7 @@
  */
 public abstract class Command implements Sendable {
   /** Requirements set. */
-  protected Set<Subsystem> m_requirements = new HashSet<>();
+  private final Set<Subsystem> m_requirements = new HashSet<>();
```

Comment:
This shouldn't be final
```suggestion
  private Set<Subsystem> m_requirements = new HashSet<>();
```